azure/azure-servicebus-amqp10/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019 Microsoft.
```

Comment:
Hi Kai.
Could you please use the header template: https://github.com/eclipse/ditto-examples/blob/master/legal/license-header.txt

The copyright holders are listed in the NOTICE file: https://github.com/eclipse/ditto-examples/blob/master/legal/NOTICE.md

Reply:
done